src/main/java/org/jenkinsci/plugins/oic/properties/LogoutQueryParameters.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.plugins.oic.properties;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import java.util.ArrayList;
+import java.util.List;
+import org.jenkinsci.plugins.oic.LogoutQueryParameter;
+import org.jenkinsci.plugins.oic.OicProperty;
+import org.jenkinsci.plugins.oic.OicPropertyDescriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Allows the addition of custom query parameters to the logout request.
+ */
+public class LogoutQueryParameters extends OicProperty {
+    @NonNull
+    private final List<LogoutQueryParameter> items;
+
+    @DataBoundConstructor
+    public LogoutQueryParameters(@CheckForNull List<LogoutQueryParameter> items) {
+        this.items = items == null ? List.of() : new ArrayList<>(items);
```

Comment:
funky...  the list should not be null - I have left a comment in the jelly that should IIUC ensure that we always get a list with at least one entry?

given that if we get an empty list we should throw some form of exception?